.travis/install_gtest.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
This install_gtest.sh is not referred from anywhere. Is it expected?

Reply:
Oh, I thought I already removed it. I'll remove.